sanket/HTML/Day3/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
we dont eed this file remove from here

Reply:
it was add automatically by vscode i will remove it